Torch/Plugins/PluginManager.cs

```diff
@@ -325,21 +325,21 @@ Assembly ResolveDependentAssembly(object sender, ResolveEventArgs args)
                     if (IsAssemblyCompatible(requiredAssemblyName, asm.GetName()))
                         return asm;
                 }
-                _log.Warn($"Could find dependent assembly! Requesting assembly: {args.RequestingAssembly}, dependent assembly: {requiredAssemblyName}");
+                if (requiredAssemblyName.Name.EndsWith(".resources", StringComparison.OrdinalIgnoreCase))
+                    return null;
+                foreach (var asm in assemblies)
+                    if (asm == args.RequestingAssembly)
+                    {
+                        _log.Warn($"Could find dependent assembly! Requesting assembly: {args.RequestingAssembly}, dependent assembly: {requiredAssemblyName}");
```

Comment:
Can we change this to "Found dependent assembly," every time I read it I think it's a typo of "Couldn't" 😛 Also, does it need to be a warning or would a debug level suffice?

Reply:
It should be couldn't x.x